javascript-big-practice/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": {
+    "json-server-auth": "^2.1.0",
+    "parcel": "^2.8.3",
```

Comment:
parcel should be in `devDependencies`. 
Read more details about the differences between dependencies and devDependencies via https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/